common/config/rush/common-versions.json

```diff
@@ -15,7 +15,8 @@
      * When someone asks for "^1.0.0" make sure they get "1.2.3" when working in this repo,
      * instead of the latest version.
      */
-    // "some-library": "1.2.3"
+    "prettier": "^1.16.4",
+    "precise-commits": "~1.0.2"
```

Comment:
Why are you using `prettier@^` but `precise-commits@~`?  We typically use `^` for all dependencies, with the exception of `typescript` (which uses minor instead of major for breaking changes), or dependencies we temporarily need to pin due to breaking changes.

Reply:
fixed.